src/main/java/com/redhat/pantheon/servlet/AsciidocRenderingServlet.java

```diff
@@ -66,7 +67,7 @@ public AsciidocRenderingServlet(
     protected void doGet(SlingHttpServletRequest request,
             SlingHttpServletResponse response) throws ServletException, IOException {
         String locale = paramValue(request, "locale", DEFAULT_MODULE_LOCALE.toString());
-        boolean draft = paramValueAsBoolean(request, "draft");
+        boolean draft = paramValueAsBoolean(request, PARAM_DRAFT);
```

Comment:
:+1: I guess while we are at this, line 69 could also use a static variable extraction.